tests/test_extension.py

```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+import pytest
+
+from scrapy.crawler import Crawler
+from scrapy import Spider
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+@pytest.fixture
+def spidermon_enabled_settings():
+    return {
+        'SPIDERMON_ENABLED': True,
+        'EXTENSIONS': {
+            'spidermon.contrib.scrapy.extensions.Spidermon': 500,
+        }
+    }
+
+
+@pytest.fixture
+def crawler(spidermon_enabled_settings):
+    crawler = Crawler(Spider, settings=spidermon_enabled_settings)
+    crawler.spider = Spider('dummy')
+    return crawler
+
+
```

Comment:
Also, this test only tests that _when calling `engine_stopped`, it'll call the proper suites_.
I think that another important test would be to check if Spidermon subscribed properly to `engine_stopped` signal from Scrapy.